### PR TITLE
feat(Prambors Radio): add button and support for the new streaming page

### PR DESCRIPTION
**Pull request by Allvaa**

### CODE REVIEW

File: websites/P/Prambors Radio/presence.ts [hunk 1]
```diff
@@ -2,7 +2,6 @@ const presence = new Presence({
   clientId: "630428033966276612"
 });
 const strings = presence.getStrings({
```
Comment (ririxidev) [inline websites/P/Prambors Radio/presence.ts:4]: Merge const.

File: websites/P/Prambors Radio/presence.ts [hunk 2]
```diff
@@ -2,7 +2,6 @@ const presence = new Presence({
...
-  play: "presence.playback.playing",
   pause: "presence.playback.paused"
 });
 let presenceData: PresenceData = {
```
Comment (ririxidev) [inline websites/P/Prambors Radio/presence.ts:7]: let -> const

**Review by ririxidev — CHANGES_REQUESTED** [review]
Heya @Allvaa!
You should check out thing(s) that I requested to change in my review. Please look at them!

Thanks,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721

**Comment by Allvaa** [comment]
done

**Review by ririxidev — APPROVED** [review]
ok, lgtm

**Review by Bas950 — APPROVED** [review]
LGTM

**Pull request merged**